src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -930,33 +856,41 @@ public void BeginGetRequestStream_CreatePostRequestThenCallTwice_ThrowsInvalidOp
         {
             HttpWebRequest request = HttpWebRequest.CreateHttp(remoteServer);
             request.Method = "POST";
+            RequestState state = new RequestState();
+            state.Request = request;
 
-            IAsyncResult asyncResult = request.BeginGetRequestStream(null, null);
+            IAsyncResult asyncResult = request.BeginGetRequestStream(RequestCallback, state);
             Assert.Throws<InvalidOperationException>(() =>
             {
                 request.BeginGetRequestStream(null, null);
             });
+            
         }
 
         [Theory, MemberData(nameof(EchoServers))]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "request stream not allowed for GET on netfx")]
```

Comment:
Another instance of a behavior difference that was not planned.  I think I added that Skip* attribute.

Historically, we have not allowed GET w/ request-body for HttpWebRequest.  This is why GetRequestStream() would throw an exception on Desktop.   For HttpClient, we relaxed this restriction in the API surface.

We could decide to fix this in .NET Core and match the same exception behavior as Desktop.